libraries/rush-lib/src/api/RushProjectConfiguration.ts

```diff
@@ -585,6 +579,17 @@ export class RushProjectConfiguration {
           operationSettingsByOperationName.set(operationName, operationSettings);
         }
       }
+
+      for (const [operationName, operationSettings] of operationSettingsByOperationName) {
+        if (operationSettings.sharding?.shardOperationSettings) {
+          // eslint-disable-next-line no-console
+          console.log(
```

Comment:
```suggestion
          console.warn(
```

Reply:
There isn't a `Terminal` instance available here?

Reply:
Not directly in the constructor, there's one available further down that I can move this to to use that if you'd like?